playbooks/preflight-check.yml

```diff
@@ -175,3 +175,31 @@
     - name: Duplicate flavors exist
       fail: msg="Duplicated flavor IDs are detected."
       when: flavor_check.stdout != ""
+
+# RLM-1372 Check NFS exsiting
+- name: Check if nfs mount exists
+  hosts: "os-infra_hosts"
```

Comment:
We need to look at also infra_hosts as os-infra_hosts is not present at older releases